README.md

```diff
@@ -826,22 +926,146 @@ The ANT contract client class exposes APIs relevant to compliant Arweave Name To
 
 #### `init({ signer })`
 
-Factory function to that creates a read-only or writeabe client. By providing a `signer` additional write APIs that require signing, like `setRecord` and `transfer` are available. By default, a read-only client is returned and no write APIs are available.
+Factory function to that creates a read-only or writeable client. By providing a `signer` additional write APIs that require signing, like `setRecord` and `transfer` are available. By default, a read-only client is returned and no write APIs are available.
```

Comment:
pretty sure its writable, not writeable, online search agrees... though may be geographically dependant.

Reply:
cscode seems to claim `writeable` 🤷

Reply:
https://en.wiktionary.org/wiki/writeable - doesn't matter - eslint picked it up